instana/instrumentation/instrument_aiohttp.py

```diff
@@ -0,0 +1,55 @@
+#Attempting to work with both versions
+
+from __future__ import absolute_import
+import opentracing.ext.tags as ext
+import instana
+import opentracing
+import wrapt
+
+
+
+try:
+    import instrument_aiohttp
```

Comment:
This should be attempting to import the aiohttp package.  Here you might have mistakenly put the module name itself (instrument_aiohttp).  If aiohttp is available then we instrument, if not then pass.